### DISCUSSION TITLE
Any plans for a bootstrap mapper?

### DISCUSSION THREAD

**Discussion by AndrewTraub** [data-driven-forms/react-forms#994] [Ideas]
Do you have any plans for a mapper for Bootstrap?

**Comment by rvsia** [comment]
Hello,

we do not plan to make it. However, we have a simple [generator](https://data-driven-forms.org/development-setup#generatingamappertemplate) in our project, that can help you to create the basic components and a project structure. 

You can fork this repository, made your own mapper and then submit it to us as a PR. Mapping basic components is very simple but it would also require tests and documentation and that can be a little bit tiring. However, we are able to assist you with all the steps and we welcome any contribution.

**Reply by Hyperkid123** [reply]
Hi @AndrewTraub.

As Richard said we do not plan to create a bootstrap mapper. Not because we don't want to. It's more a matter of we don't have enough time on our hands right now to create it. We even had to postpone the custom form state manager release because of it.

But if somebody would like to create one and contribute we would be more than happy.
